recipes/cellxgene-census/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  name: cellxgene-census
+  version: 1.16.2
+  python_min: 3.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/cellxgene_census-${{ version }}.tar.gz
+  sha256: c007031b1e42b320a191ff1f20ea4099ea935fe5416dffcd710b0e4a94c28ce1
+  patches:
+    - fix-deps.patch
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools >=64
+    - setuptools-scm >=8
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - tiledbsoma >=1.12.3,!=1.14.1
+    - anndata
+    - numpy >=1.23
+    - requests
+    - typing_extensions
+    - s3fs >=2021.06.1
+    - tiledb-py
+
+tests:
```

Comment:
Please add `pip check` to the test section

Reply:
The `pip check` is performed by default in the v1 recipe when a python test is added. See https://rattler.build/latest/reference/recipe_file/#python-tests